dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/RpcStatus.java

```diff
@@ -112,6 +112,9 @@ public static boolean beginCount(URL url, String methodName, int max) {
         max = (max <= 0) ? Integer.MAX_VALUE : max;
         RpcStatus appStatus = getStatus(url);
         RpcStatus methodStatus = getStatus(url, methodName);
+        if (methodStatus.active.get() >= max) {

```

Comment:
should we change this to a particluar check, otherwise it would be confusing:

```java
if (methodStatus.active.get() == Integer.MAX_VALUE) {
```

It would be nice if you put one comment above to explain why we need this.

Reply:
you are right, I have changed, pls verify